Tools/scripts/summarize_stats.py

```diff
@@ -350,35 +360,38 @@ def emit_execution_counts(opcode_stats, total):
             rows
         )
 
+def _emit_comparative_execution_counts(base_rows, head_rows):
+    base_data = dict((x[0], x[1:]) for x in base_rows)
+    head_data = dict((x[0], x[1:]) for x in head_rows)
+    opcodes = set(base_data.keys()) | set(head_data.keys())
+
+    rows = []
+    default = [0, "0.0%", "0.0%", 0]
+    for opcode in opcodes:
```

Comment:
Maybe give these a deterministic (and somewhat meaningful) order?

```suggestion
    for opcode in sorted(opcodes):
```

Reply:
We could, be we end up sorting by the counts a few lines below anyway...